ignite/distributed/utils.py

```diff
@@ -386,8 +386,8 @@ def broadcast(
         src: source rank. Default, 0.
         safe_mode: if True, non source input data can be ``None`` or anything (will be discarded), otherwise data
             type of the input ``tensor`` should be respected for all processes. Please, keep in mind, this mode is
-            working only for dense tensors as source input if a tensor is provided. There are additional collective
-            ops are performed before doing the broadcast and, thus, can be slower than without using this mode.
+            working only for dense tensors as source input if a tensor is provided. With ``safe_mode=False``, some
+            additional collective ops are performed before doing the broadcast so it can be faster to enable this mode.
```

Comment:
RIght now the sentence is not logically correct as `safe_mode` is `False` by default, so no need to enable it.
The whole point of the sentence about performance impact with `safe_mode=True` is to say that with `safe_mode=True` brings simplicity for the code but has a small runtime impact (a tradeoff). The way how you write the sentence leads to a confusion IMO

Reply:
How is it to say:
```
if True, non source input data can be ``None`` or anything (will be discarded), otherwise data type of the input 
``tensor`` should be respected for all processes. Please, keep in mind, this mode is working only for dense tensors as 
source input if a tensor is provided. It also leads to some additional collectives before the broadcast, making it slower 
than without using this mode. 
```

Reply:
Looks ok to me, thanks !

Reply:
Or we can keep everything unchanged but remove the second `are` in the original code.